tracker/data/processing.py

```diff
@@ -818,6 +818,6 @@ def write(content, destination, binary=False):
 
 
 def boolean_for(string):
-    if string == "False":
+    if not string or string == "False":
```

Comment:
Heh, just a few days ago I went the opposite direction: https://github.com/18F/pulse/commit/6bb4d288caed2541fbca4fb2450ab3a15655ffe9

It was a change I had made temporarily a while back when locally researching a data integrity issue and only recently committed it, so unfortunately I don't remember what bug I was fixing exactly with this. But there was an issue which this was addressing, and precision is often more helpful than convenience.

Reply:
Hmm, I actually like that better, I think I will switch to the same